tools/functional-tester/etcd-tester/tester.go

```diff
@@ -115,3 +121,77 @@ func (s *Status) setCase(c int) {
 	defer s.mu.Unlock()
 	s.Case = c
 }
+
+// checkConsistency stops the cluster for a moment and get the hashes of KV storages.
+// TODO: instead of logging to stdout, record it as an event to database.
+func (c *cluster) checkConsistency() error {
+	log.Println("canceling all stressers")
+	for _, s := range c.Stressers {
+		s.Cancel()
+		defer s.Stress()
+	}
+
+	log.Println("sleeping...")
+	time.Sleep(5 * time.Second)
+
+	log.Println("dropping all peer traffic")
+	fa := newFailureIsolateAll()
+	if err := fa.Inject(c, 0); err != nil {
+		return err
+	}
+
+	log.Println("sleeping...")
+	time.Sleep(5 * time.Second)
+
+	log.Println("getting hashes")
+	gm := make(map[string]uint32)
+	for _, u := range c.GRPCURLs {
+		conn, err := grpc.Dial(u, grpc.WithInsecure(), grpc.WithTimeout(5*time.Second))
+		if err != nil {
+			return err
+		}
+		kvc := pb.NewKVClient(conn)
+
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		resp, err := kvc.Hash(ctx, &pb.HashRequest{})
+		hv := resp.Hash
+		if resp != nil && err != nil {
+			log.Println("err:", err)
+		}
+		cancel()
+
+		gm[u] = hv
+	}
+
+	msg, cons := checkConsistency(gm)
+	log.Printf("checkConsistency result: %s (%v)", msg, cons)
+
+	log.Println("recovering all peer traffic")
```

Comment:
let's remove the failure injection in this func and then we can remove this too.